### PR TITLE
Display hash table at start

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

Display hash table for each available component in a project

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by angrybayblade — COMMENTED** [review]
table

**Review by DavidMinarsch — COMMENTED** [review]
LGTM - please add tests and add the hashes in hashes.csv too

**Review by angrybayblade — COMMENTED** [review]
hashes

File: scripts/generate_ipfs_hashes.py [hunk 1]
```diff
@@ -449,13 +450,22 @@ def update_hashes(timeout: float = 15.0) -> int:
...
                     package_hashes[key] = package_hash
+                    for result in result_list:
+                        if cast(str, result["Name"]).endswith(
+                            str(PACKAGE_TYPE_TO_CONFIG_FILE.get(package_type.value))
+                        ):
+                            package_hashes[
+                                key
+                                + f"/{PACKAGE_TYPE_TO_CONFIG_FILE.get(package_type.value)}"
+                            ] = result["Hash"]
+                            break
```
Comment (angrybayblade) [inline scripts/generate_ipfs_hashes.py:468]: Uses hash for *.yaml file generated from IPFS daemon.

File: packages/hashes.csv [hunk 2]
```diff
@@ -1,42 +1,84 @@
...
 open_aea/connections/p2p_libp2p_mailbox,QmXSJqE6gnHr5au2fUnNPSpBxv3N9Wyo6wNuzx2xytP7ar
+open_aea/connections/p2p_libp2p_mailbox/connection.yaml,QmWhZDA34ewPQroMQNiSqv5F9vcYtQhxDi2aWhFXJua3RQ
 open_aea/connections/scaffold,QmQeGqjcTvXn6sDGZCMUFTULjfHNW2RoCVN4PFeJUbbPWJ
+open_aea/connections/scaffold/connection.yaml,QmSNZgeHm1DRhhdzmsU3EKTTK4HyF4AU3dNEBaLFWF1XR7
 open_aea/contracts/scaffold,QmeqqfKY6pNLDbsCjBMxaNVdaHHd23NrpdHwYHhJ6DwqTB
+open_aea/contracts/scaffold/contract.yaml,QmZZ9g95RR9Yrbcp7ega7b4K8rFEs1r7Mp6Fr8qYwd5SZb
 open_aea/protocols/scaffold,QmSKEMPt7tC8pUukNnJzpMJrCTNxpgqVokgNBq1JToLB1t
+open_aea/protocols/scaffold/protocol.yaml,QmVJXv3czkzioSVmR3mqM58kdEEUbY9qc3xFw2V6PJbrrX
 open_aea/protocols/signing,QmXCoWUQjYLNFcybs5bpYJ8BTKtJK7KkyMWHsitdtcoLch
+open_aea/protocols/signing/protocol.yaml,QmeXkquWzNGF6qXnkz9TGsk3bmaS89CKTZHvuHTNHFWzzQ
```
Comment (angrybayblade) [inline packages/hashes.csv:82]: But there is a small issue with using a hash generated from IPFS daemon, it doesn't match with the hash generated using `IPFSHashOnly`.
```csv
open_aea/protocols/signing/protocol.yaml,QmSR9sDX5b1ZEQPHA2q997FEjSxYdzguZLhbnPazkPi4rg
fetchai/protocols/default/protocol.yaml,QmUU4hKwaheWn77dSV85352oyj3cApu53cBL3muAZDViqE
fetchai/skills/echo/skill.yaml,QmdhNmCyY3EcDrVrDAHvziKenjYXgvenGSqCtuHuT7FPeB
```

So should I use IPFSHashOnly in hashing script to generate hashes for *.yaml files?

Comment (DavidMinarsch) [reply packages/hashes.csv:82]: Not na issue anymore

**Pull request merged**